src/services/EdFi.DataManagementService.Api/Configuration/ConnectionStrings.cs

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using Microsoft.Extensions.Options;
+
+namespace EdFi.DataManagementService.Api.Configuration;
+
+public class ConnectionStrings
+{
+    public required string DatabaseConnection { get; set; }
+}
+
+public class ConnectionStringsValidator : IValidateOptions<ConnectionStrings>
+{
+    public ValidateOptionsResult Validate(string? name, ConnectionStrings options)
+    {
+        return options.DatabaseConnection == null
+            ? ValidateOptionsResult.Fail("Missing required ConnectionStrings value: DatabaseConnection")
```

Comment:
This case ends up processing differently for some reason: does not have the stack trace. It _does_ produce a 500, but it _also_ provides the proper GET response on the root endpoint.

![image](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/assets/9324390/733dfbc2-8e3a-44d5-96ab-7b89753858ea)

Reply:
Oops. I didn't have short circuit in the middleware that logs the error and returns 500. Addressed.